pages/linux/gedit.md

```diff
@@ -0,0 +1,20 @@
+# gedit
+
+> Text editor for the GNOME Desktop.
+> It can be used to create and edit all kinds of text files.
+
+- Open a text file:
+
+`gedit {{file}}`
+
+- Open multiple text files:
+
+`gedit {{file1 file2}}`
+
+- Display list of encodings:
+
+`gedit --list-encodings`
+
+- Open text file with the specified encoding:
+
+`gedit --encoding={{encoding}} {{file}}`
```

Comment:
```suggestion
`gedit --encoding={{UTF-8}} {{path/to/file}}`
```
Let's also use a concrete example here instead of repeating the word "encoding".